RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -93,31 +93,25 @@ import RevenueCat
     #if DEBUG
 
     convenience init(
-        hasSubscriptions: Bool = false,
-        areSubscriptionsFromApple: Bool = false
+        hasActiveProducts: Bool = false,
+        hasAppleEntitlement: Bool = false
     ) {
         self.init(customerCenterActionHandler: nil)
-        self.hasSubscriptions = hasSubscriptions
-        self.subscriptionsAreFromApple = areSubscriptionsFromApple
+        self.hasActiveProducts = hasActiveProducts
+        self.hasAppleEntitlement = hasAppleEntitlement
         self.state = .success
     }
 
     #endif
 
     func loadHasSubscriptions() async {
```

Comment:
It might be a good idea to rename this function since the model no longer has a property named `hasSubscriptions` and it's also loading non-subscriptions now

```suggestion
    func loadHasActivePurchases() async {
```